testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RegisterWithUserProfileTest.java

```diff
@@ -284,12 +284,12 @@ public void testAttributeGuiOrder() {
         );
         Assert.assertTrue(
                 driver.findElement(
-                        By.cssSelector("form#kc-register-form > div:nth-child(4) > div:nth-child(2) > input#password")
+                        By.cssSelector("form#kc-register-form > div:nth-child(4) > div:nth-child(2) > div > input#password")
```

Comment:
The following should give the same result, considering that an `id` must be globally unique.
```suggestion
                        By.cssSelector("#password")
```